mPowerSDK/mPowerSDK/Startup/APHAppDelegate.m

```diff
@@ -35,38 +35,89 @@
 #import "APHAppDelegate.h"
 #import "APHProfileExtender.h"
 #import "APHDataKeys.h"
+#import "APHLocalization.h"
 
 static NSString *const kMyThoughtsSurveyIdentifier                  = @"mythoughts";
 static NSString *const kEnrollmentSurveyIdentifier                  = @"EnrollmentSurvey";
 static NSString *const kStudyFeedbackSurveyIdentifier               = @"study_feedback";
 
-
-
 /*********************************************************************************/
 #pragma mark - Initializations Options
 /*********************************************************************************/
-static NSString *const kStudyIdentifier             = @"studyname";
-static NSString *const kAppPrefix                   = @"studyname";
 static NSString *const kConsentPropertiesFileName   = @"APHConsentSection";
 
 static NSString *const kVideoShownKey = @"VideoShown";
 
-
 static NSString *const kJsonScheduleStringKey           = @"scheduleString";
 static NSString *const kJsonTasksKey                    = @"tasks";
 static NSString *const kJsonScheduleTaskIDKey           = @"taskID";
 static NSString *const kJsonSchedulesKey                = @"schedules";
-static NSString *const kAppStoreLink                    = @"https://appsto.re/us/GxN85.i";
 
-@interface APHAppDelegate ()
+static NSString *const kAppStoreLink                    = @"http://apple.co/1FO7Bsi";
```

Comment:
(I checked, no, it doesn't. But the original https://appsto.re/us/GxN85.i link seems to still work--was this change a result of merging from the app store repo? Or was there some other reason behind it?)